site/index.html

```diff
@@ -76,6 +76,54 @@ <h2>Try it!</h2>
 
 <br>
 
+<div class="features">
+   <div class=" container ">
+      <h2>Why Haskell?</h2>
+      <div class=" row ">
+         <div class=" span6 col-md-6">
+            <h3>Rapid prototyping</h3>
+            <p>Start small and explore new ideas to find your market fit. Haskell is a compact language with a large ecosystem of open-source libraries and tools
+               to get you started and help you growing.
+               The strong type system makes it easy to write programs that work, both large and small, even when you don't like writing types &mdash; the compiler will figure it out.
+            </p>
+         </div>
+         <div class=" span6 col-md-6">
+            <h3>Scale with ease</h3>
+            <p>From 1 to 100k users, and more. Haskell offers one of the best support for async, concurrent and parallel programming like few other languages can.
+               It can take advantage of multi-core processors with ease thanks to garbage collection, green threads and immutability everywhere.
+               Additionally, you can use advanced streaming libraries for efficient data processing
+               &mdash; all while keeping complexity low as your user base and code base grow.
+            </p>
+         </div>
+      </div>
+      <div class=" row ">
+         <div class=" span6 col-md-6">
+            <h3>Maintain with confidence</h3>
+            <p>Add complex features or refactor large parts of your code base reliably. Haskell has one of the strongest type systems on the market to prevent mistakes
+               before they reach production, saving on total software lifetime costs.&mdash;“If it compiles, it works” and "Fearless refactoring" are common proverbs among Haskellers.
+            </p>
+         </div>
+         <div class=" span6 col-md-6">
+            <h3>Bleeding edge</h3>
+            <p>Get a view into the latest programming innovations. Haskell has its roots in programming language research, has already influenced many popular languages and continues to do so.
+               Experiment with new programming techniques like Effect Systems, Functional Reactive Programming (FRP), Linear Types
+               and more related to Dependent Types for even better reliability when you can tolerate some immaturity;
+               or stay with stable best practices&mdash;Haskell supports both.
```

Comment:
I don't think it really makes sense to talk about "better reliability" and "immaturity" together. I think willingness to "experiment" already covers "immaturity" so we don't need to mention it.

"and more related to Dependent Types" doesn't seem grammatical, but it's also unnecessary, so I suggest removing it.

```suggestion
            <p>Get a view into the latest programming innovations. Haskell has its roots in programming language research, has already influenced many popular languages and continues to do so.
               Experiment with new programming techniques like Effect Systems, Functional Reactive Programming (FRP), Linear Types
               and Dependent Types;
               or stay with stable best practices&mdash;Haskell supports both.
```

Reply:
If you are happy with the changes I propose please make them. Then I'm happy to recommend the PR to the rest of the committee.